docker/Dockerfile.ci

```diff
@@ -110,7 +110,7 @@ RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest
 
 # Add code stub last
 COPY . /root/code/garage
-ENV PYTHONPATH=$PYTHONPATH:/root/code/garage
+RUN ["/bin/bash", "-c", "pip install -e /root/code/garage"]
```

Comment:
You need to `source activate garage` before you do this, or else you're using the system pip and not the conda pip.

e.g.
```Dockerfile
RUN ["/bin/bash", "-c", "source activate garage && pip install -e /root/code/garage"]
```

I'm surprised the CI passed, but it's probably because `WORKDIR` happens to be right in the repo root.